### PR TITLE
Delete Socialblade.com-falsemixed.xml

### PR DESCRIPTION
**Pull request by koops76**
#10595

### CODE REVIEW

**Comment by cschanaj** [comment]
remove reference from [`Socialblade.com.xml#L2`](https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/Socialblade.com.xml#L2)

**Comment by koops76** [comment]
@cschanaj I can't. I deleted the fork.

File: src/chrome/content/rules/Socialblade.com-falsemixed.xml [hunk 1]
```diff
@@ -1,13 +0,0 @@
-
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://forums.socialblade.com/ => https://forums.socialblade.com/: (6, 'Could not resolve host: forums.socialblade.com')
-
-	See also Socialblade.com.xml
--->
-<ruleset name="Socialblade.com (mixed content)" platform="mixedcontent" default_off='failed ruleset test'>
```
Comment (J0WI) [inline src/chrome/content/rules/Socialblade.com-falsemixed.xml]: This host is still up.

Comment (koops76) [inline src/chrome/content/rules/Socialblade.com-falsemixed.xml]: No, it is not:
```
>nslookup forums.socialblade.com carl.ns.cloudflare.com
Server:  carl.ns.cloudflare.com
Address:  173.245.59.106

Name:    forums.socialblade.com

>nslookup forums.socialblade.com eva.ns.cloudflare.com
Server:  eva.ns.cloudflare.com
Address:  173.245.58.114

Name:    forums.socialblade.com
```

Comment (J0WI) [inline src/chrome/content/rules/Socialblade.com-falsemixed.xml]: `forums` is down, but `socialblade.com` itself is still up (see `Socialblade.com.xml`).

File: src/chrome/content/rules/Socialblade.com-falsemixed.xml [hunk 2]
```diff
@@ -1,13 +0,0 @@
-
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://forums.socialblade.com/ => https://forums.socialblade.com/: (6, 'Could not resolve host: forums.socialblade.com')
-
-	See also Socialblade.com.xml
```
Comment (J0WI) [inline src/chrome/content/rules/Socialblade.com-falsemixed.xml]: Obsolete reference

File: src/chrome/content/rules/Socialblade.com-falsemixed.xml (hunk 1 above)
Comment (koops76) [inline src/chrome/content/rules/Socialblade.com-falsemixed.xml]: I didn't touch `Socialblade.com.xml`.

File: src/chrome/content/rules/Socialblade.com-falsemixed.xml (hunk 2 above)
Comment (koops76) [inline src/chrome/content/rules/Socialblade.com-falsemixed.xml]: I can't remove the reference to the deleted file from Socialblade.com.xml unless I will create another pull request.

Comment (J0WI) [inline src/chrome/content/rules/Socialblade.com-falsemixed.xml]: Okay, could you open a new PR?

Comment (koops76) [inline src/chrome/content/rules/Socialblade.com-falsemixed.xml]: @J0WI #11053

**Pull request closed without merging**